codeworld-api/src/CodeWorld.hs

```diff
@@ -115,23 +115,12 @@ module CodeWorld
     Event (..),
 
     -- * Debugging
-    trace,
-
-    -- * FMIDUE
-    move,
-    print,
-    rotate,
-    path,
-    color,
-    scale,
-    Points,
-    String
+    trace
   )
 where
 
 import Prelude ((<>))
 import CodeWorld.Color
 import CodeWorld.EntryPoints
 import CodeWorld.Event
-import CodeWorld.Picture
-import CodeWorld.FMIDUE
\ No newline at end of file
+import CodeWorld.Picture
```

Comment:
>  I see such changes as the above a lot in your commits ...

If you are only referring to this repository, then it is probably because there is no `.editorconfig` in place. The formatters I use usually respect this config.

Reply:
I meant in general (if such an `.editorconfig` does not exist).

Reply:
(But introducing a `.editorconfig` in this repository would be fine by me.)